react-native/react/native/remote-component-loader.js

```diff
@@ -52,24 +50,39 @@ class RemoteStore {
   }
 }
 
+function getQueryVariable(variable) {
```

Comment:
this is the only place its used in our actual codebase vs in the html files. i'm exploring a different way for us to deal with the html so i might be able to do it